CommandHelper/LocalPackages/commands/chars/joker.yml

```diff
@@ -20,6 +35,8 @@ permissions:
 powers:
 - /items
 - /effects
+- /run 2
+- /jump 2
```

Comment:
I think the joker who is beating batman up is not the same joker, probs should be a different character.